modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/v1_1/AxisOrderTest.java

```diff
@@ -175,11 +175,13 @@ public void testGetFeatureWithEastNorthAxisOrderFilter() throws Exception {
         source.getFeatures(query).features();
         BBOX filter = (BBOX) wfs.getRequest().getFilter();
 
-        // filter coordinates are NOT inverted
-        assertEquals(4623055.0, filter.getMinX(), 0.0);
-        assertEquals(815134.0, filter.getMinY(), 0.0);
-        assertEquals(4629904.0, filter.getMaxX(), 0.0);
-        assertEquals(820740.0, filter.getMaxY(), 0.0);
+        // filter coordinates ARE inverted (EPSG:3857 is EAST/NORTH, so if we ask for NORTH/EAST,
```

Comment:
Uh ok, this one is interesting. So the BBOX is built as follows:
ff.bbox("the_geom", 4623055.0, 815134.0, 4629904.0, 820740.0, "EPSG:3857")
and indeed 3857 is projected and not flipped... so this is indeed a fix, the axis order should be preserved, no flipping.

Reply:
So ... the new tests are ok, right?

Reply:
Yep, they look ok, I'm weireded out by the old version of the test verifying that a projected system caused the filter to have inverted axis...

Reply:
I do not remember right now how the correct axis order is checked and where it is documented. For example EPSG:2393 (N-E), EPSG:3067 (E-N), and EPSG:3047 (N-E).